src/main/java/Duke.java

```diff
@@ -1,10 +1,32 @@
+/**
+ * Responsible for running the Duke application, and responding to user.
+ * Initializes and terminates the application.
+ */
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    private TaskList tasks;
```

Comment:
Maybe name this to taskList instead of using tasks to avoid using plural form since there is only one taskList?